rdm-tools.php

```diff
@@ -2189,24 +2172,22 @@ function getSpawnData($args) {
       echo json_encode(array('spawns' => null, 'status'=>'Error: no points!'));
       return;
     }
-
     if (is_numeric($args->nest_migration_timestamp) && (int)$args->nest_migration_timestamp == $args->nest_migration_timestamp) {
       $ts = $args->nest_migration_timestamp;
     } else {
       $ts = 0;
     }
     $binds[] = $ts;
 
-    
-      
+
     if (is_numeric($args->spawn_report_limit) && (int)$args->spawn_report_limit == $args->spawn_report_limit && (int)$args->spawn_report_limit != 0) {
       $limit = " LIMIT " . $args->spawn_report_limit;
     } else {
       $limit = '';
-    }    
-  
+    }
+
     $sql_spawn = "SELECT pokemon_id, COUNT(pokemon_id) as count FROM rdmdb.pokemon WHERE " . $points_string . " AND first_seen_timestamp >= ? GROUP BY pokemon_id ORDER BY count DESC" . $limit;
```

Comment:
this sql-qry needs to be fixed ...
instead of `rdmdb.pokemon` use `pokemon`
this was wrong all along ... kudoo to f.e. DrKendall